recipes/server_source.rb

```diff
@@ -165,6 +166,8 @@
   service_name 'zabbix-server' if node['init_package'] == 'systemd'
   supports :status => true, :start => true, :stop => true, :restart => true
   action [:start, :enable]
+  notifies :run, 'execute[pagerDuty_syslink]', :immediately
```

Comment:
Do we need to add these blocks here? can't it be kicked off from our personal zabbix cookbook?